nmostesting/Config.py

```diff
@@ -119,6 +119,10 @@
 # This must match the domain name used for certificates in HTTPS mode
 DNS_DOMAIN = "testsuite.nmos.tv"
 
+# The host name of the mocks used by the Controller tests when they are run securely. This should be consistent
+# with the hosts entries in the host running the test suite and the host running the Controller under test.
+MOCKS_HOSTNAME = "mocks.testsuite.nmos.tv"
```

Comment:
Why or why not should this just be built from `DNS_DOMAIN` ?

Also, may need to be `nmos-mocks.local` or `mocks.<DNS_DOMAIN>`, depending on `DNS_SD_MODE` ?

Reply:
Yes good point - I will investigate `DNS_SD_MODE` configuration further.

Reply:
Mock hostname is now calculated based on DOMAIN_NAME and DNS_SD_MODE. It's been removed from the Config.py.